student.rb

```diff
@@ -0,0 +1,13 @@
+require './person'
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(classroom, age, name, parent_permission)
```

Comment:
- [ ] This line is giving error, when I'm trying to create a student object with no name because it is supposed to take the default name but here, you have specified the default `name` or `parent_permission` value, Please fix this.

```ruby
def initialize(age, classroom, name = "Unknown", parent_permission = true)
```